src/components/proxy-middleware/helpers/harObectCreator.js

```diff
@@ -12,9 +12,14 @@ const createHarHeaders = (request_headers_obj) => {
   return headers;
 };
 
-const createHarQueryStrings = (path) => {
-  // TODO -> http://www.softwareishard.com/blog/har-12-spec/#queryString
-  return [];
+const createHarQueryStrings = (query_params) => {
+  return Object.keys(query_params).map( query => {
+    const val = query_params[query]
+    return {
+      "name" : query,
+      "value": Array.isArray(val) ? val[0] : val // because these are passed as arrays from upstream
```

Comment:
If there are multiple values available, then we should map over all the values and created multiple name/value objects, instead of using a single one. 

Also val if always array from upstream. `Array.isArray` seems redundant